metron-deployment/roles/quick_dev/tasks/main.yml

```diff
@@ -15,23 +15,50 @@
 #  limitations under the License.
 #
 ---
-#
-# Workaround for Kafka not starting
-# Fire off async start followed by
-# Sync start -execution will pause until
-# final start completes.
-#
-- name: Start the ambari cluster - no wait
-  ambari_cluster_state:
+- name: Delete the Metron Components from Ambari
+  ambari_service_state:
     host: "{{ groups.ambari_master[0] }}"
     port: "{{ ambari_port }}"
     username: "{{ ambari_user }}"
     password: "{{ ambari_password }}"
     cluster_name: "{{ cluster_name }}"
-    cluster_state: started
-    wait_for_complete: False
+    state: deleted
+    component_name: "{{ item }}"
+    component_host: "{{ inventory_hostname }}"
+  with_items:
+    - METRON_ENRICHMENT_MASTER
+    - METRON_INDEXING
+    - METRON_PARSERS
+
+- name: Remove the Metron packages
+  package:
+    name: "{{ item }}"
+    state: absent
+  with_items:
+    - metron-common
+    - metron-data-management
+    - metron-parsers
+    - metron-enrichment
+    - metron-indexing
+    - metron-elasticsearch
+
+- name: Re-install the Metron Packages via Ambari
```

Comment:
No, I want them to exist there and they do. It simulates the old run_enrichment_role task by re-installing the Metron rpms and reloads configs, but leaves everything else alone.